### DISCUSSION TITLE
[bug] Academy - Consultant - PIM presentation: bad links

### DISCUSSION THREAD

**Discussion by nycoala** [pimcore/pimcore#16543] [Q&A / Support] [answered]
## Actual behaviour
The "Related Academy Courses" links all target the current page because they have not href value.
> [Extending Data Objects](https://pimcore.com/academy/course/Pimcore-PIM~132588/chapter-132597)
> [Working with Objects](https://pimcore.com/academy/course/Pimcore-PIM~132588/chapter-132597)
> [Working with Objects (Advanced)](https://pimcore.com/academy/course/Pimcore-PIM~132588/chapter-132597)

The related HTML source:
```html
<li>
    <span class="fa fa-external-link"></span>
    <a href="" target="_blank">Extending Data Objects</a>
</li>
<li>
    <span class="fa fa-external-link"></span>
    <a href="" target="_blank">Working with Objects</a>
</li>
<li>
    <span class="fa fa-external-link"></span>
    <a href="" target="_blank">Working with Objects (Advanced)</a>
</li>
```

## Expected behaviour
These links should redirect to
- [Extending Data Objects](https://pimcore.com/academy/course/Extending-Data-Objects~13495/overview)
- [Working with Objects](https://pimcore.com/academy/course/Working-with-Objects~12797/overview)
- [Working with Objects (Advanced)](https://pimcore.com/academy/course/Working-with-Objects-Advanced~359381/overview)

## Steps to reproduce
- Go to the "Consultant / Pimcore PIM - Presentation" page: https://pimcore.com/academy/course/Pimcore-PIM~132588/chapter-132597
- Check the "Related Academy Courses" links.

**Comment by fashxp** [comment, marked as answer]
thx for sharing, is fixed

**Reply by nycoala** [reply]
Thank you.
You're fast!
